cmd/promtool/testdata/rules_extrafields.yml

```diff
@@ -0,0 +1,31 @@
+# This is the rules file.
```

Comment:
I couldn't tell at a glance whether there are extra fields here - could that be in the comment?